client/src/pages/Login.js

```diff
@@ -17,14 +17,14 @@ import { useQuery } from "../utils/hooks.js";
 import { theme, mq } from "../constants/theme";
 
 // ICONS
+import { Visibility, VisibilityOff } from '@material-ui/icons';
```

Comment:
Let's not use third party icons. Please get the icons from Figma. Instructions ares in the README. You can also review how we use icons currently from several components.